Digital Clock/Button434x64.qml

```diff
@@ -0,0 +1,53 @@
+import QtQuick
```

Comment:
Questo mi sembra uguale a `Button190x64`, con solo piccole modifiche, dovute probabilmente al fatto che li hai scritti in due tempi diversi. Li possiamo unire insieme?